dvc/progress.py

```diff
@@ -127,18 +133,42 @@ def __exit__(self, typ, value, tbck):
         self._lock.release()
 
 
-class ProgressCallback(object):
-    def __init__(self, total):
+class CheckoutProgressCallback(object):
+    TOO_LONG_CHECKOUT_SECONDS = 60.0
```

Comment:
Maybe SLOW_CHECKOUT_TIMEOUT ?

Reply:
Why don't we make it a few seconds - 5 or 10?